src/Domain/FundRepository.php

```diff
@@ -274,5 +281,8 @@ protected function doUpdateFromSf(SalesforceReadProxy $proxy, bool $withCache):
 
         $fundData = $this->getClient()->getById($fundId, $withCache);
         $proxy->setName($fundData['name'] ?? '');
+        // Not sure I'm entirely convinced anything but tests is using this for Funds, but mirroring what we do with
+        // 'name' for now just in case. Maybe we can make the fn throw instead? Needs more attention later ideally.
+        $proxy->setSlug($fundData['slug'] ?? '');
```

Comment:
yes it doesn't look like any prod code calls `\MatchBot\Domain\SalesforceReadProxyRepository::updateFromSf` except where it knows that that the `SalesforceReadProxyRepository` is actually a `CampaignRepository`. Might be worth just making this function generate an error if called since it'd be alarming to be wrong about that. And then after that we could make it throw or better just move updateFromSf down from `SalesforceReadProxyRepository` to `CampaignRepository` and delete this function.

Reply:
We use `\MatchBot\Domain\FundRepository::pullForCampaign` instead to update funds.